src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/Inputs/SARIF1019.RuleIdMustBePresentAndConsistent_Valid.sarif

```diff
@@ -0,0 +1,79 @@
+{
+  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.5.json",
+  "version": "2.1.0",
+  "runs": [
+    {
+      "tool": {
+        "driver": {
+          "name": "Sarif Functional Testing"
+        }
+      },
+      "logicalLocations": [
```

Comment:
>logicalLocations [](start = 7, length = 16)

To demonstrate your fix, you don't need logical locations. In fact, you don't need any locations at all. (A `result` object is not required to have a `locations` property.) Just remove this, and remove the `locations` property from all the results.

Reply:
ok!